app/api/tests/test_api.py

```diff
@@ -715,6 +715,96 @@ def test_disallows_non_annotation_approver_to_approve_and_disapprove_labels(self
     def doCleanups(cls):
         remove_all_role_mappings()
 
+class TestCommentListAPI(APITestCase):
+
+     @classmethod
+     def setUpTestData(cls):
+         cls.project_member_name = 'project_member_name'
+         cls.project_member_pass = 'project_member_pass'
+         cls.another_project_member_name = 'another_project_member_name'
+         cls.another_project_member_pass = 'another_project_member_pass'
+         cls.non_project_member_name = 'non_project_member_name'
+         cls.non_project_member_pass = 'non_project_member_pass'
+         create_default_roles()
+         cls.project_member = User.objects.create_user(username=cls.project_member_name,
+                                                       password=cls.project_member_pass)
+         another_project_member = User.objects.create_user(username=cls.another_project_member_name,
+                                                           password=cls.another_project_member_pass)
+         User.objects.create_user(username=cls.non_project_member_name, password=cls.non_project_member_pass)
+
+         main_project = mommy.make('SequenceLabelingProject', users=[cls.project_member, another_project_member])
+         main_project_doc = mommy.make('Document', project=main_project)
+         cls.comment = mommy.make('Comment', document=main_project_doc, text='comment 1', user=cls.project_member)
```

Comment:
assigned value not used

Reply:
The cls.comment variable is used in line 770 and 776.

Reply:
nevermind.